src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <!-- search_big -->
+    <form
+      class="search-bar big"
+      data-qa="big"
+    >
+      <img
+        src="/src/images/Search.svg"
+        alt="search"
+      />
```

Comment:
Redundant, need to fix it
```suggestion
```
<img width="487" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/4850c5ac-2aad-4d06-b0ca-fe7390afa9ad">